### PR TITLE
Animation Function

### PR DESCRIPTION
**Pull request by cslotboom**
Add the animation function, as well as some tests of the function in 2D and 3D.

### CODE REVIEW

File: openseespy-pip/openseespy/postprocessing/Get_Rendering.py [hunk 1]
```diff
@@ -120,48 +120,48 @@ def readODB(*argv):
 	First record all the output data and then read it instantly for plotting.
 	"""
     
-	ModelName = argv[0]
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/Get_Rendering.py:123]: Are these indentation changes? It is only in readODB() and not in createODB().

File: openseespy-pip/openseespy/postprocessing/Get_Rendering_Animations.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
 # If yes, force inline, interactive backend for Matplotlib.
 import sys
 import matplotlib
+import openseespytools.model
```
Comment (u-anurag) [inline openseespy-pip/openseespy/postprocessing/Get_Rendering_Animations.py]: This will cause an import error.

**Review by u-anurag — CHANGES_REQUESTED** [review]
Looks good. Two comments,
1) Are we consistent with the indentation changes?
2) Line 22 "import openseespytools.model" in Get_Rendering_Animation will cause an error.

File: openseespy-pip/openseespy/postprocessing/Get_Rendering_Animations.py (hunk 2 above)
Comment (cslotboom) [reply openseespy-pip/openseespy/postprocessing/Get_Rendering_Animations.py]: Should be now removed.

File: openseespy-pip/openseespy/postprocessing/Get_Rendering.py (hunk 1 above)
Comment (cslotboom) [reply openseespy-pip/openseespy/postprocessing/Get_Rendering.py:123]: I had some issues with the tabs on my end, - the script won't run on my end unless it's all spaces or all tabs. 
It should be converted back to tabs now.

**Comment by cslotboom** [comment]
Hey, the changes have been addressed - I converted a bunch of my spaces back to tabs.

**Review by u-anurag — APPROVED** [review]

**Pull request merged**